examples/dnsdoctor/libxt_dnsdoctor.lua

```diff
@@ -0,0 +1,28 @@
+local luaxt = require("luaxt")
+local family = luaxt.family
+
+local function noop() end
+
+local function dnsdoctor_init(par)
+	local target_ip = "10.1.2.3"
+	local target = 0
+	target_ip:gsub("%d+", function(s) target = target * 256 + tonumber(s) end)
+
+	local src_ip = "10.1.1.2"
+	local src = 0
+	src_ip:gsub("%d+", function(s) src = src * 256 + tonumber(s) end)
```

Comment:
you can require `socket.inet` and just use `aton()`

Reply:
That library is available in the kernel space, right ?

Reply:
yeah.. you're right.. I'd leave it as is.. but could you create an issue to move the utilities from `socket.inet` such as `aton()` to a separate script to be shared between user space and kernel? I think it would be a good first issue for new contributors =).

Reply:
Done, created #169